eslintrc.json

```diff
@@ -21,6 +21,8 @@
 
   "rules": {
     "accessor-pairs": 2,
+    "arrow-body-style": [2, "as-needed"],
```

Comment:
`as-needed` is the default it looks like: http://eslint.org/docs/rules/arrow-body-style